notebooks/MAST/spectraldb_and_specviz/spectraldb_and_specviz.ipynb

```diff
@@ -0,0 +1,919 @@
+{
```

Comment:
<p>I think <code>Spectrum1D</code> is designed to work robustly if a user provides the input in any number of ways, either read in from a FITS file or manually specifying flux and wavelength as we do here.  It ignores most metadata from an input FITS file.   The <code>Specviz</code> tool does use more metadata when the input is a FITS file, but it also designed to work well when it is passed <code>Spectrum1D</code> objects or numpy arrays, when that metadata is not available.   For the purposes of this notebook with the goals of searching for spectra and quickly visualizing the results, most science use cases should be covered.   </p><p> </p><p>The <code>fluxErr</code> is the default standard error from the original x1d FITS files, corresponding to the <code>FLUX_ERROR</code> column in the <code>EXTRACT1D</code> extension.   There's more information here, https://jwst-pipeline.readthedocs.io/en/latest/jwst/data_products/science_products.html#extracted-1-d-spectroscopic-data-x1d-and-x1dints.  They do provide variances as well, which are also returned by the retrieve API call.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>